src/test/java/org/mule/test/extension/file/common/ConcurrentLockTestCase.java

```diff
@@ -148,4 +163,107 @@ protected PathLock createLock(Path path) {
     @Override
     public void changeToBaseDir() {}
   }
+
+  @Test
+  public void concurrentUriLock() throws Exception {
```

Comment:
The two tests are the same, but changing tryLock for tryUriLock, may be you could parameterize the test.